dashboard/config/scripts/netsim.script

```diff
@@ -3,5 +3,19 @@ description_short ""
 description ""
 hidden 'true'
 
+stage 'Sending Bits'
+level 'blockly:NetSim:unit1_lesson04'
```

Comment:
my same comment on lesson number.  see infinity.script for example of using named levels.  it made things way easier when it came to shuffling lesson order which inevitably happened.